README.md

```diff
@@ -7,15 +7,62 @@ A set of load tests for OCM's clusters-service, based on vegeta.
 Compile using `make` and run as a simple binary:
 
 ```sh
-./ocm-api-load --token=<OCM_TOKEN>   \
-    --output-path="/path/to/output/" \
-    --gateway-url=<api-gateway-url>  \
-    --duration-in-min=1              \
+./ocm-api-load --config-file /path/to/config
 ```
 
-## CLI Parameters
+if your `config-file` is named `ocm-api-load.yaml` and it in the same
+path as your binary, it will be autodetected and you could run by just calling it:
```

Comment:
@chentex Do you mind adding to the `README.md` the names of the tests to be specified in the `config` file (And that they need to correspond to names in `/tests/tests.go`)?